share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-neoverse_n1/spack.yaml

```diff
@@ -0,0 +1,61 @@
+spack:
+  view: false
+
+  definitions:
+  - compiler_specs:
+    - gcc
+    - gettext
+
+  - compiler_target:
+    - '%gcc@7.3.1 target=aarch64'
+
+  - optimized_configs:
+    - gromacs
+    # - lammps
+    # - mpas-model
+    - openfoam
+    - palace
+    # - py-devito
+    - quantum-espresso
+    # - wrf
+
+  - optimized_libs:
+    - mpich
+    - openmpi
+
+  specs:
+  - matrix:
+    - - $compiler_specs
+    - - $compiler_target
+  - $optimized_configs
+  - $optimized_libs
+
+
+  mirrors: { "mirror": "s3://spack-binaries/develop/aws-pcluster-aarch64" }
```

Comment:
I will change the mirror names to match the pipelines in another commit after the pipelines ran though.